### DISCUSSION TITLE
Help with (docker) deployment

### DISCUSSION THREAD

**Discussion by sdevogel** [directus/directus#17969] [Archive]
Hi,

Im quite new to docker and have the hardest time to figure out how to deploy it. I would like to deploy the directs to render.com without making any changes to it and use it as a CMS. Render.com doesn't support docker compose and requires a dockerfile to build the container. 

Any help is much appreciated!
Thanks in advance 🙏

**Comment by rijkvanzanten** [comment]
Based on the examples they provide for other Docker based tools, you should be able to make a new Dockerfile with just:

```Dockerfile
FROM directus/directus:9.23.4
```
